projects/demo/src/modules/experimental/card/examples/16/index.less

```diff
@@ -0,0 +1,3 @@
+.map {
+    border-radius: 0.5rem;
```

Comment:
I kinda like it more when it's not rounded but that's up to you :)

Reply:
fixed